1.test.near.json

```diff
@@ -0,0 +1 @@
+{"account_id":"1.test.near","public_key":"ed25519:AdKaReGjRqPcUTfKah4k1Fv1af6seLj4FtpkxLsFfgQL","secret_key":"ed25519:3qR2uxAN7SUfi9HziHGejXRcprNK2YkYKnpKjtzm7rvgxkXW5MDMbKZMAEsxcnFJgtb2W7QgAChc1cvr2Da8b6eY"}
```

Comment:
I don't think this is needed anymore?
It was originally for a test network we running (which can be shutdown now if it wasn't @mikedotexe) - now this is useless with EVM on Betanet

Reply:
good catch, I'll do that tomorrow